client.go

```diff
@@ -32,11 +32,12 @@ func isPacketConn(c net.Conn) bool {
 
 // A Conn represents a connection to a DNS server.
 type Conn struct {
```

Comment:
conn is used a lot... it should grow and grow and we've already added a new tsig thing here.